src/EA.Weee.Web/Areas/Admin/ViewModels/AddAatf/AddFacilityViewModelBase.cs

```diff
@@ -15,6 +15,34 @@ public abstract class AddFacilityViewModelBase : FacilityViewModelBase
 
         public IEnumerable<short> ComplianceYearList => new List<short> { 2019 };
 
+        public static new IEnumerable<string> ValidationMessageDisplayOrder => new List<string>
+        {
```

Comment:
Can we add a unit test to check this property and that they are in this order :)

Reply:
Added